annotations/annotators.py

```diff
@@ -153,11 +154,13 @@ def render_display(self, context={}):
     def get_context(self):
         resource = self.get_resource()
         request = self.context.get('request')
+        content = self.get_content(resource)
+        detect  = chardet.detect(content)
         return {
             'text': self.text,
             'textid': self.text.id,
             'title': 'Annotate Text',
-            'content': self.get_content(resource),
+            'content': content.decode(detect['encoding']).encode('utf-8'), 
```

Comment:
let's put a comment here that says we need to do this guessing because of a bug in Giles (it returns everything with a utf-8 header, even if the text is not in utf-8)

Reply:
@jdamerow  Done